pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -49,3 +58,21 @@ func (a *ApicastOptionsProvider) setReplicas() {
 	a.apicastOptions.ProductionReplicas = int32(*a.apimanager.Spec.Apicast.ProductionSpec.Replicas)
 	a.apicastOptions.StagingReplicas = int32(*a.apimanager.Spec.Apicast.StagingSpec.Replicas)
 }
+
+func (a *ApicastOptionsProvider) setStagingLabels(image string) map[string]string {
+	labels := helper.MeteringLabels("apicast-staging", helper.ParseVersion(image), helper.ApplicationType)
+	labels["app"] = a.apicastOptions.AppLabel
+	labels["threescale_component"] = "apicast"
```

Comment:
We'll end up with duplicated key/values between templates and operator for all components. I'd be nice to not have them duplicate. Can you think of some way to avoid that? I guess constants in the components package would be a way. Not sure whether we would have lots of them though. What do you think?

Reply:
done

Reply:
duplication of templates is not a big deal. They will soon be removed